designcompose/src/main/java/com/android/designcompose/InteractionState.kt

```diff
@@ -34,7 +34,7 @@ import com.android.designcompose.serdegen.View
 // to a specific component instance instead of all of them, such as changing to a different variant.
 private fun getInstanceIdWithKey(instanceId: String, key: String?): String {
     var varKey = instanceId
-    if (key != null) varKey += key
+     if (key != null) varKey += key
```

Comment:
Accidental space

Reply:
That's... an indication that I didn't get something right, since spotless/ktfmt should be catching that